pontoon/base/utils.py

```diff
@@ -653,3 +659,14 @@ def build_translation_memory_file(creation_date, locale_code, entries):
         u'\n\t</body>'
         u'\n</tmx>'
     )
+
+
+def add_newrelic_context(**kwargs):
+    try:
+        for key, value in kwargs.items():
+            newrelic.agent.add_custom_parameter(key, value)
+    except AttributeError:
+        # If newrelic-admin isn't used, for example in local development,
+        # then newrelic.agent will not be set.
+        log.warning('Unable to add context to newrelic')
```

Comment:
This might get a little bit too verbose over time once we add more context?

Reply:
That should only happen in environments where we do not have newrelic enabled, and as far as I know, that means only dev. I guess we can do with excessive amounts of logs in dev?

Reply:
OK, let's keep it.